properties/P000133.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: P000133
+slug: linearly_orderable
+name: Linearly orderable
+aliases:
+  - Linearly ordered topological space
+  - LOTS
+refs:
+  - wikipedia: Order_topology
+    name: Order topology
+  - mr: '3728284'
+    name: Topology (Munkres)
+  - mr: MR2048350
+    name: General Topology (Willard)
+  - mr: MR1039321
+    name: General Topology (Engelking, 1989)
+---
+
+A space whose topology can be induced by a total order on the underlying set.
```

Comment:
A reader should be able to get the definition just from reading this without having to look up sources. So need to describe what that topology induced by a total order is.